server/szurubooru/api/tag_category_api.py

```diff
@@ -73,6 +73,11 @@ def update_tag_category(
         tag_categories.update_category_color(
             category, ctx.get_param_as_string("color")
         )
+    if ctx.has_param("order"):
+        auth.verify_privilege(ctx.user, "tag_categories:edit:order")
+        tag_categories.update_category_order(
+            category, ctx.get_param_as_int("order")
+        )
```

Comment:
Copy this code diff into the `create_tag_category` method above this one. Current code is unable to set the order upon category creation, only when editing existing ones.